client/src/components/Common/NearbyHeader.tsx

```diff
@@ -0,0 +1,60 @@
+
+import { View, Text, StyleSheet, Dimensions, Image } from 'react-native'
+import React from 'react'
+
+interface CounterProps {
+    count: number;
+  }
+
+export const HeaderCount: React.FC<CounterProps> = ({ count }) => {
```

Comment:
Rename this to NearbyHeader to match the file name!

Reply:
Done!